include/API/errors.h

```diff
@@ -47,6 +47,7 @@ enum class ErrorCategory {
   QSSLinkArgumentNotFoundWarning,
   QSSLinkInvalidPatchTypeError,
   QSSControlSystemResourcesExceeded,
+  QSSCompilerNonExistingCommandError,
```

Comment:
```suggestion
  QSSTargetUnsupportedOperation,
```

Perhaps it would make more sense to refer to this as an `Operation` to be consistent with MLIR and other non-gate functionality

along with

```
  case ErrorCategory::QSSCompilerNonExistingCommandError:
    return "An unsupported operation on a target was used";
```

Reply:
Done.